bridge/client/app/project-board/project-board.component.spec.ts

```diff
@@ -156,6 +155,24 @@ describe('ProjectBoardComponent', () => {
     });
   });
 
+  it('should not show notification indicator', () => {
+    expect(fixture.nativeElement.querySelector('.notification-indicator.notification-indicator-absolute')).toBeFalsy();
```

Comment:
Change description e.g. `should not show notification indicator on component creation`

Reply:
done